graphql-cli/src/test/resources/expectedGenCode/client/clientAndAPIKeysConfig/config_types.bal

```diff
@@ -14,38 +14,40 @@
 // specific language governing permissions and limitations
 // under the License.
 
+import ballerina/http;
+
 # Client configuration details.
 @display {label: "Connection Config"}
 public type ConnectionConfig record {|
     # Configurations related to client authentication
-    BearerTokenConfig auth;
+    http:BearerTokenConfig auth;
     # The HTTP version understood by the client
-    HttpVersion httpVersion = HTTP_V2_0;
+    http:HttpVersion httpVersion = http:HTTP_2_0;
     # Configurations related to HTTP/1.x protocol
-    ClientHttp1Settings http1Settings = {};
+    ClientHttp1Settings http1Settings?;
```

Comment:
Have we missed the `http:` here?

Reply:
No. Used a custom record to do a modification in http record.